src/Akka.Hosting/AkkaConfigurationBuilder.cs

```diff
@@ -190,47 +191,55 @@ public AkkaConfigurationBuilder StartActors(ActorStarter starter)
         }
 
         public AkkaConfigurationBuilder WithCustomSerializer(
-            string serializerIdentifier, IEnumerable<Type> boundTypes, Func<ExtendedActorSystem, Serializer> serializerFactory)
+            string serializerIdentifier, IEnumerable<Type> boundTypes,
+            Func<ExtendedActorSystem, Serializer> serializerFactory)
         {
             var serializerRegistration = new SerializerRegistration(serializerIdentifier,
                 boundTypes.ToImmutableHashSet(), serializerFactory);
-            _serializers.Add(serializerRegistration);
+            Serializers.Add(serializerRegistration);
             return this;
         }
 
-        internal void Build()
+        internal void Bind()
```

Comment:
Separate `Build` and `Bind` - need to have `Bind` run outside the context of the `IServiceProvider` construction in order to make sure that all of the registrations are added up front. We may not even really need the `Build` method any longer.

Reply:
Removed the `Build` method as it's no longer needed.